doc/source/webapp_ui/terminal/index.rst

```diff
@@ -0,0 +1,10 @@
+==========
+Terminal
+==========
+
+.. image:: ../images/terminal.gif
+   :align: center
+
+Use the Terminal page to execute shell commands. Provided as a convenience for installing dependencies, executing scripted workflows, and so on.
```

Comment:
This seems like a pretty significant security exposure. Do we want to put a warning in the installation instructions?

I assume there's no way to disable the terminal functionality in the UI?

Reply:
Terminal feature does not introduce any extra security exposure as anything that can be done in the terminal, can be done via an electron which can be run in "local" executor of the hosted covalent server. For instance, one can submit an electron to run any shell command in the server even without terminal now. All terminal does is expose it easier for administration. Note that covalent OS is recommended for only single user setup in its current form and its up to the user to secure the architecture they are hosting it on by making VPC etc..

Reply:
Understood. I guess if you're giving someone access to your HPC hardware, they'd better be someone you trust. It's been a long time since I've approached computing from a cooperative perspective. In working with everyday network security, paranoia comes with the territory. 😀